docs/CSBR.md

```diff
@@ -848,7 +849,7 @@ Private Keys corresponding to Root Certificates MUST NOT be used to sign Certifi
 
 ### 6.2.6 Private key transfer into or from a cryptographic module
 
-For Certificates transported outside of a Signing Service's secure infrastructure, the CA or Signing Service MUST require, by contract, each Subscriber to generate their own Private Key and protect the Private Key in accordance with [Section 6.2.7.4](#6274-subscriber-private-key-protection-and-verification).
+The CA or Timestamp Authority SHALL NOT transfer Private Keys from a cryptographic module to a Subscriber.

```

Comment:
I'm not sure why this is necessary. It is strange to read that we disallow a CA or Timestamp Authority Private Key to be transferred to a Subscriber... I think the idea is to restrict ANY transfer of those types of keys outside a cryptographic module.

Recommended change:

The CA or Timestamp Authority SHALL NOT transfer Private Keys from a cryptographic module to a destination that does not meet the requirements of [Section 6.2.7.4](#6274-subscriber-private-key-protection-and-verification).

Reply:
First, this looks like a typo so "The CA and Timestamp Authority" should be "The Timestamp and Signing Service" SHAL NOT ... The original requirement stated requirements for both the CA and the Signing Service, so the proposal was to be consistent. Perhaps we should state "Subscriber Private Keys".